functions/src/modules/pm/groups/updateGroup.ts

```diff
@@ -1,5 +1,49 @@
-import { ExpressFunction } from "../../../@types";
+import type { ExpressFunction } from "../../../@types";
+import { firebaseApp } from "../../../config/firebaseConfig";
+import {
+  expressErrorHandlerFactory,
+  requestBodyTypeValidator,
+} from "../../../helpers";
+import { uasPermissionSwitch } from "../../../systems/uas";
 
-const replaceMe: ExpressFunction = (req, res, next) => {};
+const updateGroup: ExpressFunction = (req, res, next) => {
+    uasPermissionSwitch({
+        organizer: {accepted: validateOrganizer },
+    }) (req, res, next)
+};
+const validateOrganizer: ExpressFunction = (req, res, next) => {
+    const validationRules: ValidatorObjectRules = {
+      type: "object",
+      rules: {
+        prizeGroupId: { rules: ["string"] },
+        prizeGroupName: { rules: ["string"] },
+        prizeGroupDescription: { rules: ["string"] },
+        prizeGroupOrder: {
+          rules: [
+          {
+              type: "array",
+              rules: ["string"],
+          },
+          ],
+        },
+          },
+    };
+    requestBodyTypeValidator(req, res, next)(validationRules, execute);
+}; 
 
-export default replaceMe;
+const execute: ExpressFunction = (req, res, next) => {
+  const errorHandler = expressErrorHandlerFactory(req, res, next);
+
+  firebaseApp
+    .firestore()
+    .collection("PrizeCategoryGroup")
```

Comment:
Can we call this `prizeGroups` to stick with the naming convention from other modules?
```suggestion
    .collection("prizeGroups")
```